update-openssl-include-guards

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Should this script live in util?  or even the tools repo?

Reply:
It’s intended to thrown away after use.

Reply:
IOW I‘ll discard the commit when merging.

Reply:
Okay, I suspected as much.  Better to be sure.